Tests/scripts/runTests.py

```diff
@@ -103,6 +106,7 @@ def RunTestCase(asn1, acn, behavior, expErrMsg):
             PrintFailed("coverage failed. (less than 100%)")
             sys.exit(1)
```

Comment:
At first glance, this sounds like an issue with the compiler bundled in Mac OS - I can't reproduce this in either Debian or Arch.